frontend/packages/console-shared/src/utils/pod-ring-utils.ts

```diff
@@ -44,6 +50,27 @@ export const podRingLabel = (obj: K8sResourceKind, canScale: boolean): PodRingLa
   };
 };
 
+export const usePodScalingAccessStatus = (
+  obj: K8sResourceKind,
+  resourceKind: K8sKind,
+  pods: ExtPodKind[],
+  enableScaling?: boolean,
+  impersonate?,
+) => {
+  const [editable, setEditable] = React.useState(false);
+  React.useEffect(() => {
+    checkPodEditAccess(obj, resourceKind, impersonate)
+      .then((resp: SelfSubjectAccessReviewKind) => setEditable(resp.status.allowed))
```

Comment:
Getting this error in Console

![Screenshot from 2019-12-16 18-11-40](https://user-images.githubusercontent.com/2561818/70907778-bde9f680-202f-11ea-9e0a-6794328bb5b8.png)

Reply:
@vikram-raj I'm not able to reproduce this error. Can you provide more information regarding this?

Reply:
@sahil143 This error shows up when I import Operator backed service.

Reply:
I am able to see this error too. `resp.status` is a optional attribute so add checks to prevent cannot read property of null error.

Reply:
@karthikjeeyar @vikram-raj Added a check. Can you confirm that this error is resolved. https://github.com/openshift/console/pull/3746/commits/e14a5a7b140b29c2fc9e26a77cb118db4ffb3cc9

Reply:
@sahil143 Now, I do not see this error.